test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -295,32 +521,132 @@ define([
+                            Object.keys(batchJob.originalJobs).length
+                        );
+                        expect(Object.keys(batchJob.currentJobs).length).toEqual(allRows.length);
+                    });
+
+                    it('has the correct original rows', () => {
+                        allRows = Array.from(container.querySelectorAll('tbody tr'));
+                        // make sure all rows are present in batchJob.originalJobs
+                        const allIds = allRows.map((row) => {
+                            return row.id;
+                        });
+                        const expectedIds = Object.keys(batchJob.originalJobs).map((job_id) => {
+                            return `job_${job_id}`;
+                        });
+                        expect(allIds).toEqual(jasmine.arrayWithExactContents(expectedIds));
+                    });
+
+                    Object.keys(batchJob.currentJobs).forEach((job_id) => {
+                        it(`has the correct row content for job ${job_id}`, function () {
+                            this.input = batchJob.jobsById[job_id];
+                            this.job = this.input.retry_parent
+                                ? batchJob.jobsById[this.input.retry_parent]
+                                : batchJob.jobsById[job_id];
+                            this.row = container.querySelector(`#job_${this.job.job_id}`);
+                            // check the row content
+                            _checkRowStructure(this.row, this.job, this.input);
+                        });
+                    });
+                });
+                describe('table init with job info', () => {
+                    beforeEach(function () {
+                        container = document.createElement('div');
+                        this.job = JSONcopy(paramTestsJobArray[0]);
+                        this.jobManager = new JobManager({
+                            model: makeModel(paramTestsJobArray),
+                            bus: Runtime.make().bus(),
+                        });
+                        this.jobManager.addListener('job-info', [
+                            'job_update_test',
+                            'generic_retry_parent',
+                        ]);
+                    });
+
+                    paramTests.forEach((test) => {
+                        it(
+                            'initialises correctly using the job params: ' +
+                                JSON.stringify(test.input),
+                            async function () {
+                                // add the job params
+                                this.jobManager.model.setItem('exec.jobs.params', {
+                                    job_update_test: test.input.job_params[0],
+                                });
+                                this.jobStatusTableInstance = await createStartedInstance(
```

Comment:
the table is drawn when the widget is started, so we have to get all the param information in place before running `this.jobStatusTableInstance.start()`